kymatio/scattering1d/backend/numpy_backend.py

```diff
@@ -110,26 +81,35 @@ def unpad(x, i0, i1):
     """
     return x[..., i0:i1]
 
+def concatenate(arrays):
+    return np.stack(arrays, axis=-2)
 
 
+def rfft(x):
+    real_check(x)
+    return scipy.fftpack.fft(x)
 
-def concatenate(arrays):
-    return np.stack(arrays, axis=-2)
+
+def irfft(x):
+    complex_check(x)
+    return np.real(scipy.fftpack.ifft(x))
```

Comment:
Does `np.real` output a view? If so, OK. If not, can we rather have `scipy.fftpack.ifft(x).real`, because that for sure is a view

Reply:
the docs seem to imply that np.real and .real are the exact same thing in the example. I'll change it to `.real`